code/parse/sexp.cpp

```diff
@@ -22475,6 +22475,15 @@ void sexp_show_subtitle(int node)
 					if (n != -1)
 					{
 						post_shaded = is_sexp_true(n);
+						n = CDR(n);
+
+						if (n != -1)
+						{
+							auto percent = eval_num(n, is_nan, is_nan_forever);
+							if (is_nan || is_nan_forever)
+								return;
```

Comment:
Silently skipping over the SEXP entirely with `return` seems a little extreme to me. How about ignoring the bad value and issuing a `Warning()` that the provided line height factor is invalid? Unless we're concerned that using the default value here could result in something worse that ill-formatted text, although bad formatting would be a nice obvious indicator what the FREDder did wrong.

Reply:
I don't think `return` is extreme here; it's consistent with other usage in sexp.cpp.  NANs are almost always treated as invalid values that don't do anything.

Reply:
Also, the percent could conceivably be any number, even negative.  It may produce unusual results, but it's still valid, and I figured a modder should be able to use it if there was a need for it.